### ISSUE TITLE
Update of pip

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/RobotFramework_AIO#232]
Maybe it's worth to avoid this repeated hint by updating pip in next AIO release:

```
[notice] A new release of pip is available: 23.3.2 -> 24.0
[notice] To update, run: D:\RobotFramework\python39\python.exe -m pip install --upgrade pip
```

**Comment by huavanthong** [comment]
Hi @HolQue , 

Based on my check, we've already updated the latest pip package on AIO. I think that the issue with your output log occurred  because it redirected to a different location for the pip package.

Could you help me to check the output log for the command below on your machine?
```
D:\RobotFramework\python39\python.exe -m pip --version
```

Thank you and best regards,
Thong Hua

**Comment by test-fullautomation** [comment]
Hi  @HolQue ,
can you please check.
Thank you,
 Thomas

**Comment by HolQue** [comment]
Hi Thong,

on my machine it's:

pip 23.3.2 from D:\RobotFramework\python39\lib\site-packages\pip (python 3.9)

**Comment by test-fullautomation** [comment]
Hi Holger,
the problem is that you have to run our setup scripts (cloneall?) from beginning.
With rebasing of existing repos there will be no update of the python. This will be only one time created at setup time of the whole development environment.
Our build and SelfTests install always all new at new virtual machines, therefore there the latest version will be used.
Thank you,
 Thomas

**Comment by HolQue** [comment]
Hi Thomas,

this I do not understand. I use the RobotFramework AIO installer and expect to get the current state of all components installed.

**Comment by test-fullautomation** [comment]
Hi @HolQue ,
O.K. got the point! 
@huavanthong : when I install RobotFramework AIO new with our windows and/or deb installer, then I expect also the latest pip version. Can you please check this.
Thank you,
 Thomas

**Comment by huavanthong** [comment]
Hi Thomas, Holger

The AIO package've been already installed the latest pip package. 
It can be found at
https://github.com/test-fullautomation/RobotFramework_AIO/blob/a859e7f7039c3b0b82baf1a7b092c553a966a2aa/install/install.sh#L302

Therefore, we will get the latest pip package in the next AIO release.
However, I noticed that the Linux platform has not been updated to the latest version yet, so I would like to create a pull request https://github.com/test-fullautomation/RobotFramework_AIO/pull/244 for this.  

Please take it for your info.

Thank you and best regards,
Thong Hua

**Comment by test-fullautomation** [comment]
very good!
I already merged #244
